src/Env/index.js

```diff
@@ -156,6 +157,31 @@ class Env {
     return _.get(process.env, key, defaultValue)
   }
 
+  /**
+   * Get value for a given key from the `process.env`
+   * object or throw an error if the key does not exist.
+   *
+   * @method getOrFail
+   *
+   * @param  {String} key
+   *
+   * @return {Mixed}
+   *
+   * @example
+   * ```js
+   * Env.getOrFail('MAIL_PASSWORD')
+   * ```
+   */
+  getOrFail (key) {
+    const val = _.get(process.env, key)
+
+    if (_.isUndefined(val)) {
+      throw GE.RuntimeException.incompleteConfig(`process.env.${key}`, '.env')
```

Comment:
Let's use the `missingEnvKey` exception here as discussed on https://github.com/adonisjs/adonis-framework/issues/847#event-1610545001